housing-prices-intermediate/README.md

```diff
@@ -0,0 +1,273 @@
+  - [Step 7: Roll back to the previous commit](#step-7-roll-back-to-the-previous-commit)
+  - [Step 8: Update our dataset](#step-8-update-our-dataset)
+
+## Housing Prices Dataset
+
+The housing prices dataset used for this example is a reduced version of the original [Boston Housing Datset](https://www.cs.toronto.edu/~delve/data/boston/bostonDetail.html), which was originally collected by the U.S. Census Service. We choose to focus on three features of the originally dataset (RM, LSTST, and PTRATIO) and the output, or target (MEDV) that we are learning to predict.
+|Feature| Description|
+|---|---|
+|RM |       Average number of rooms per dwelling|
+|LSTAT |    A measurement of the socioeconomic status of people living in the area|
+|PTRATIO |  Pupil-teacher ratio by town - approximation of the local education system's quality|
+|MEDV |     Median value of owner-occupied homes in $1000's|
+
+Sample:
+|RM   |LSTAT|PTRATIO|MEDV|
+|-----|----|----|--------|
+|6.575|4.98|15.3|504000.0|
+|6.421|9.14|17.8|453600.0|
+|7.185|4.03|17.8|728700.0|
+|6.998|2.94|18.7|701400.0|
+
+## Prerequisites
+
+Before you can deploy this example you need to have the following components:
+
+1. A clone of this Pachyderm repository on your local computer. (could potentially include those instructions)
+2. A Pachyderm cluster - You can deploy a cluster on [PacHub](hub.pachyderm.com) or deploy locally as described [here](https://docs.pachyderm.com/latest/getting_started/).
+
+Verify that your environment is accessible by running `pachctl version` which will show both the `pachctl` and `pachd` versions.
+```bash
+$ pachctl version
+COMPONENT           VERSION
+pachctl             1.11.0
+pachd               1.11.0
+```
+
+## Pachyderm Pipelines
+
+We'll deploy each stage in our ML process as a Pachyderm pipeline. Organizing our work into pipelines allows us to keep track of artifacts created in our ML development process. We can extend or add pipelines at any point to add new functionality or features, while keeping track of code and data changes simultaneously.
+
+### Data Analysis Pipeline
+The data analysis pipeline creates a pair plot and a correlation matrix showing the relationship between features. By seeing what features are positively or negatively correlated to the target value (or each other), it can helps us understand what features may be valuable to the model.
+<p align="center">
+  <img width="300" src="images/pairplot-1.png">
+  <img width="300" src="images/corr_matrix-1.png">
+</p>
+
+### Split Pipeline
+Split the input `csv` files into train and test sets. As we new data is added, we will always have access to previous versions of the splits to reproduce experiments and test results. 
+
+Both the `split` pipeline and the `data_analysis` pipeline take the `csv_data` as input but have no dependencies on each other. Pachyderm is able to recognize this and can run each pipeline simultaneously, scaling them horizontally. 
```

Comment:
```suggestion
Both the `split` pipeline and the `data_analysis` pipeline take the `csv_data` as input but have no dependencies on each other. Pachyderm is able to recognize this. It can run each pipeline simultaneously, scaling each horizontally.
```